k2vr-installer-gui/Tools/Utils.cs

```diff
@@ -20,40 +20,58 @@ public static bool IsDirectoryEmpty(string path)
         public static bool EnsureSteamVrClosed()
         {
             Logger.Log("Checking if SteamVR is open...", false);
-            foreach (Process process in Process.GetProcesses())
+
+            //Check if SteamVR is running
+            if (Process.GetProcesses().FirstOrDefault((Process proc) => proc.ProcessName == "vrserver" || proc.ProcessName == "vrmonitor") == null)
+            {
+                return true;
```

Comment:
Should log "Done!" before this return (and the other return on the second check, if it is true) for consistency